reference=HJD.chr.fasta
query=cowpea-Cell1.Clean.fq.gz
prefix=winn.ont
directory=merylDBO
modle=map-ont

meryl count k=15 output ${directory} ${reference}
meryl print greater-than distinct=0.9998 ${directory}> ${prefix}_repetitive_k15.txt
winnowmap -W ${prefix}_repetitive_k15.txt -ax ${modle} ${reference} ${query} -t 64 > ${prefix}.sam
samtools view -@ 64 -bS ${prefix}.sam | samtools sort -@ 64 - -o ${prefix}.bam
samtools index ${prefix}.bam
rm ${prefix}.sam

# Obtain the identifiers of reads mapping to the gap region with IGV or samtools.

seqtk telo -m CCCTAAA HJD.hifi.fastq.gz > hifi.telo.stat
seqtk telo -m CCCTAAA ONT.Clean.fq.gz > ont.telo.stat
awk '{print $1}' hifi.telo.stat > hifireads.bed
awk '{print $1}' ont.telo.stat > ontreads.bed
seqtk subseq HJD.hifi.fastq.gz hifireads.bed > hifi.telo.reads.fasta
seqtk subseq ONT.Clean.fq.gz ontreads.bed > ont.telo.reads.fasta

# use hifi reads or ont reads
# The hifi.bam file was generated by mapping reads to a genome assembly whose chromosomes are slightly truncated and lack telomeric regions.
samtools view -b -f 4 -F 256 hifi.bam > unmapped.primary.bam
samtools fasta unmapped.primary.bam > unmapped_reads.fa
cat hifi.telo.reads.fasta unmapped_reads.fa > hifi.re.fasta
/public/home/weichuanzheng/software/hifiasm-0.20.0/hifiasm -t 48 -o HJDr --telo-m CCCTAAA --dual-scaf hifi.re.fasta